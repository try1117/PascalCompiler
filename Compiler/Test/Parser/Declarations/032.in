program test;
var
  b : integer = 1.23 * 456;
begin
end.
