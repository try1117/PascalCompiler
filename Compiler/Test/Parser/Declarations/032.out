Lexical exception in position (3,27) - Incompatible types, expected Integer but found Double
